wire-schema/src/test/java/com/squareup/wire/schema/PrunerTest.kt

```diff
@@ -420,6 +420,42 @@ class PrunerTest {
     assertThat((pruned.getType("SomeFieldOptions") as MessageType).field("c")).isNull()
   }
 
+  @Test @Ignore("Need to address #1327.")
+  fun includeExtensionOfOptionPrunesOtherExtensionFields() {
+    val schema = RepoBuilder()
+        .add("lecture.proto", """
+             |package squareup.options.lecture;
+             |
+             |import "google/protobuf/descriptor.proto";
+             |
+             |extend google.protobuf.FieldOptions {
+             |  optional bool relevant = 22301;
+             |  optional bool unrelevant = 22302;
```

Comment:
grammar nazi nit: `irrelevant`

Reply:
might be old english https://www.lexico.com/en/definition/unrelevant